src/Schemas/Format.xsd

```diff
@@ -0,0 +1,758 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema http://www.w3.org/2001/XMLSchema.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xs="http://www.w3.org/2001/XMLSchema" xml:lang="en">
+  <xs:element name="Configuration">
+    <xs:annotation>
+      <xs:documentation>Represents the top-level element of a formatting file.</xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:all>
+        <xs:element minOccurs="0" name="DefaultSettings" maxOccurs="1" type="DefaultSettings">
+          <xs:annotation>
+            <xs:documentation>Optional element. Defines common settings that apply to all the views of the formatting file.</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element minOccurs="0" name="SelectionSets" maxOccurs="1" type="SelectionSets">
+          <xs:annotation>
+            <xs:documentation>Optional element. Defines the common sets of .NET Framework objects that can be used by all views of the formatting file.</xs:documentation>
```

Comment:
`.NET Framework` is specifically not `.NET Core`.  Since this applies to both, I would suggest just changing all occurrences of `.NET Framework` to simply `.NET`